.github/workflows/ci.yaml

```
@@ -47,17 +47,14 @@ jobs:
               run: |
                   sudo echo "127.0.0.1" `hostname` | sudo tee -a /etc/hosts
 
+            - name: Configure debug logging
+              if: runner.debug == '1'
```

@danielpeintner Would something like this work for you? :)

```suggestion
            # Enables the output of log messages from the Node.js debug module
            # for workflow runs that are being retried with "debug logging"
            # enabled.
            #
            # See https://github.com/eclipse-thingweb/node-wot/pull/1204 for
            # more context regarding the rationale of this step.
            - name: Configure debug logging
              if: runner.debug == '1'
```

---

Looks good!